deepfence_ui/app/scripts/components/integration-view/advance-filter-modal.js

```diff
@@ -63,9 +63,12 @@ export const AdvancedFilterModalContent = ({
 
   const filterValues = filters || initialFilters;
 
+  // eslint-disable-next-line array-callback-return
+  const nodeFiltersOptions = nodeFilters && nodeFilters.filter(item => { if (item.label !== 'CloudTrail') return item;});
```

Comment:
I don't get this, filter callback is supposed to return boolean.

Reply:
I added this because we don't want to show the cloudtrail filter on the advanced filters popup since the cloudtrail filter comes through the enumerate API. There is a check now that hides the advanced filters button when the user selects cloudtrail under resource. But it would still show the filter in the advanced filters pop-up when the user selects any other resource. This line removes the cloudtrail filter just for the popup